### PR TITLE
Add Matcha Dark Sea theme (org.gtk.Gtk3theme.Matcha-dark-sea)

### PR DESCRIPTION
**Pull request by Mek101**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://github.com/vinceliuice/Matcha-gtk-theme/issues/153
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by Mek101** [comment]
bot, build org.gtk.gtk3theme.matcha-dark-sea

**Comment by Mek101** [comment]
bot, build org.gtk.Gtk3theme.Matcha-dark-sea

**Comment by Mek101** [comment]
bot, build org.gtk.Gtk3theme.Matcha-dark-sea

File: org.gtk.Gtk3theme.Matcha-dark-sea.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{
+    "id": "org.gtk.Gtk3theme.Matcha-dark-sea",
+    "branch": "3.22",
+    "runtime": "org.freedesktop.Platform",
+    "build-extension": true,
+    "sdk": "org.freedesktop.Sdk",
+    "runtime-version": "19.08",
```
Comment (bilelmoussaoui) [inline org.gtk.Gtk3theme.Matcha-dark-sea.json:7]:
```suggestion
    "runtime-version": "20.08",
```

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```
Comment (bilelmoussaoui) [inline LICENSE:1]: is a license needed here?

Comment (Mek101) [reply LICENSE:1]: I put it in there to clarify the license of the repository itself. I can remove it if it doesn't fit though

File: org.gtk.Gtk3theme.Matcha-dark-sea.json (hunk 1 above)
Comment (Mek101) [reply org.gtk.Gtk3theme.Matcha-dark-sea.json]: I'm also bumping to 20.08 all the others

File: README.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Matcha-dark-sea on Flathub
```
Comment (barthalion) [inline README.md:1]: No need to have documentation here. Flatpak will auto-install matching theme from Flathub if it's available.

**Comment by barthalion** [comment]
It's fine to keep the license around if you insist but in general we don't do that.

**Comment by Mek101** [comment]
I put the readme and the license mostly because for whoever happens to stumble on the repos like I did at the beginning: without an explicit license the repo is in a bit of legally gray area for contributions and for others to download and modify, the readme it's more to use as a reference to the original source (and I agree to reduce it)

**Pull request closed without merging**